test/fuzzing/hb-draw-fuzzer.cc

```diff
@@ -16,10 +16,10 @@ struct _draw_data_t
 
 #include <cstdio>
 static void
-_move_to (hb_draw_funcs_t *dfuncs, void *draw_data_,
+_move_to (hb_draw_funcs_t * /* dfuncs */, void *draw_data_,
```

Comment:
Instead of commenting it out, use `HB_UNUSED` macro.

Reply:
```suggestion
_move_to (hb_draw_funcs_t *dfuncs HB_UNUSED, void *draw_data_,
```

Reply:
Done.
Thank you for the tip !